cmd/security-file-token-provider/res/docker/configuration.toml

```diff
@@ -0,0 +1,19 @@
+[SecretService]
+Scheme = "https"
+Server = "edgex-vault"
+ServerName = "edgex-vault"
+Port = 8200
+# CaFilePath = "EdgeXFoundryCA.pem"
```

Comment:
Since this is commented out in both versions of `configuration.toml` would recommend to just delete it.

Reply:
Changed both to `CaFilePath = '/tmp/edgex/secrets/ca/ca.pem'`

Reply:
This is to align with #1716